test/util/genesis/genesis.go

```diff
@@ -137,36 +161,56 @@ func (g *Genesis) AddValidator(val Validator) error {
 	return nil
 }
 
-func (g *Genesis) Accounts() []Account {
-	return g.accounts
-}
-
-func (g *Genesis) Export() (*coretypes.GenesisDoc, error) {
-	addrs := make([]string, 0, len(g.accounts))
-	pubKeys := make([]cryptotypes.PubKey, 0, len(g.accounts))
-	gentxs := make([]json.RawMessage, 0, len(g.genTxs))
+// TODO improve this function to imply that we're just adding one validator to make it deterministic
```

Comment:
[question] is this TODO still applicable? If yes, should it go inside the function body so that it doesn't show up as Go docs for this function?